bedrock/firefox/templates/firefox/firstrun/firstrun-quantum.html

```diff
@@ -4,23 +4,44 @@
 
 {% add_lang_files "firefox/new/quantum" %}
 
-{% extends "firefox/firstrun/base-pebbles.html" %}
+{% from "macros.html" import fxa_email_form, fxa_link_fragment with context %}
+
+{% extends "firefox/base-protocol.html" %}
+
+{# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
+{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
 
 {% block body_id %}firstrun-quantum-onboarding{% endblock %}
 
 {% block page_title %}
   {{ _('Welcome to Firefox') }}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
   {{ css_bundle('firefox_firstrun_quantum') }}
 {% endblock %}
 
+{% block gtm_page_id %}data-gtm-page-id="/firefox/firstrun/"{% endblock %}
+
+{% block extrahead %}
+  {% if switch('firefox-update-notification-modal', ['en-US']) %}
+    {{ css_bundle('firefox-update-notification-firstrun-whatsnew') }}
+  {% endif %}
+{% endblock %}
+
+{# Bug 1387984 #}
+{% block update_notification %}
+  {% if switch('firefox-update-notification-modal', ['en-US']) %}
+    {% include 'includes/firefox-update-instructions.html' %}
```

Comment:
Passing note: We've never ported this notification to Protocol, so I'm not sure how well it will play here without some testing?

Reply:
Good point, I'll test it. I think I can try installing an old Firefox, or else just fake it locally.

Reply:
This was a good catch. The update banner was expecting styles from sandstone base, which were obviously missing from Protocol so it was unstyled. Added the styles to the bundle for this page and it looks good, despite the mix of frameworks. 

At some point we'll need to decide the fate of this banner and either kill it or redesign it for Protocol.